plugins/event_notification/providers/http/lib/KDispatchHttpNotificationEngine.class.php

```diff
@@ -182,13 +184,18 @@ public function sendHttpRequest(KalturaHttpNotificationTemplate $httpNotificatio
 		$errCode = $curlWrapper->getErrorNumber();
 		$errMessage = $curlWrapper->getError();
 
+		$header_size = curl_getinfo($curlWrapper->ch, CURLINFO_HEADER_SIZE);
+		$headers = substr($results, 0, $header_size);
```

Comment:
since you are adding CURLOPT_RETURNTRANSFER if this the headers will already be stored in infra/general/KCurlWrapper.class.php:64

Reply:
Actually I already checked this in my debugging. In this case, we don't have the $headers because we need to request to store them(using the method here -KCurlWrapper.class.php:847), and the only method using it is getHeader()(KCurlWrapper.class.php:453) but in our case, we need also the body in the result. In order to do so I needed to change some methods in  KCurlWrapper and I decided not to touch so sensitive area which I can do but here it's only info we need to log so logic change will be overkill(I think). what do you think?